tracer/src/Datadog.Trace/Debugger/Symbols/SymbolsUploader.cs

```diff
@@ -136,9 +136,9 @@ private void UnRegisterToAssemblyLoadEvent()
             AppDomain.CurrentDomain.AssemblyLoad -= CurrentDomain_AssemblyLoad;
         }
 
-        private async void CurrentDomain_AssemblyLoad(object? sender, AssemblyLoadEventArgs args)
```

Comment:
we need an analyzer to detect this pattern.

Reply:
Worth a look 👀  https://www.nuget.org/packages/Microsoft.VisualStudio.Threading.Analyzers#readme-body-tab

Reply:
ok, looked at it, and it complains about a _lot_ of things 😂 https://github.com/DataDog/dd-trace-dotnet/tree/andrew/ci/add-async-analyzer